ephios/core/views/event.py

```diff
@@ -50,6 +50,16 @@ def get_queryset(self):
                 start_time=Min("shifts__start_time"),
                 end_time=Max("shifts__end_time"),
             )
+            .annotate(
+                can_change=Case(
+                    When(
+                        id__in=get_objects_for_user(self.request.user, ["core.change_event"]),
```

Comment:
@felixrindt I'm not sure if it would be smarter to force the evaluation of the get_objects_for_user() queryset in advance and I'm not sure how to interpret the SQL explain here

Reply:
As far as I understand - given that the database executes a sensible strategy - there would be no benefit in evaluating that queryset before.